demisto_sdk/commands/common/content/objects/pack_objects/classifier/classifier.py

```diff
@@ -38,10 +139,92 @@ def upload(self, client: demisto_client) -> bool:
         """
         return client.import_classifier(file=self.path)
 
+    def validate(self):
+        return self.is_valid_classifier()
+
+    def is_valid_classifier(self):
+        """Checks whether the classifier is valid or not.
+
+        Returns:
+            bool. True if classifier is valid, else False.
+        """
+        return all([
+            self.is_valid_version(),
+            self.is_valid_from_version(),
+            self.is_valid_to_version(),
+            self.is_to_version_higher_from_version(),
+        ])
+
+    def is_valid_version(self):
+        # type: () -> bool
+        """Base is_valid_version method for files that version is their root.
+
+        Return:
+            True if version is valid, else False
+        """
+        if self.get('version') != DEFAULT_VERSION:
+            error_message, error_code = Errors.wrong_version(DEFAULT_VERSION)
+            if self.base.handle_error(error_message, error_code, file_path=self.path,
+                                      suggested_fix=Errors.suggest_fix(str(self.path))):
+                return False
+        return True
+
+    def is_valid_from_version(self):
+        """Checks if from version field is valid.
+
+        Returns:
+            bool. True if from version field is valid, else False.
+        """
+        is_valid = True
```

Comment:
why are we not consistent with the other validation functions? we dont have that `is_valid` var